tts/Object.def.lua

```diff
@@ -290,14 +290,14 @@ function Object.getRotationValue() end
 ---@return tts__Vector
 function Object.getScale() end
 
----@shape tts__Object_SnapPoint
----@field position tts__Vector Position of the snap point. The position is relative to the entity's center (a local position).
----@field rotation tts__Vector Rotation of the snap point. The rotation is relative to the entity's rotation (a local rotation).
----@field rotation_snap boolean If the snap point is a "rotation" snap point.
 
 --- Returns a table of sub-tables, each sub-table representing one snap point.
----@return tts__Object_SnapPoint[]
-function Object.getSnapPoints() end
+---@return tts__SnapPoint[]
+function Object.getSnapPoints() return {} end
```

Comment:
Missed this `return` statement in my initial review. Just a reminder it's here.

Reply:
Sorry, I missed this in the initial review and think our messages/commits cross. If you could remove this return statement, that'd be appreciated.

Reply:
Fixed.